www/on/npm/index.html.spt

```diff
@@ -1,9 +1,129 @@
+from gratipay.utils import icons
 [---]
-title = "Search npm"
+banner = manager = "npm"
 suppress_sidebar = True
-website.redirect('/')  # nothing here yet
+npackages, Npackages = website.db.one('''
```

Comment:
I think the naming here could be better. These values are only being pulled so that we can show the `x packages out of y are on Gratipay` - might be better to keep them in a single dict so that a confusion doesn't arise between `npackages`, `Npackages` and `packages_for_claiming`. 

How about - ```npm_stats = website.db.one('''select () as total_packages, () as claimed_packages''', return_as=dict)```, followed by ```n=npm_stats.total_packages, N=npm_stats.claimed_packages``` (in the template)?